src/zopflipng/zopflipng_lib.h

```diff
@@ -76,4 +125,6 @@ int ZopfliPNGOptimize(const std::vector<unsigned char>& origpng,
     bool verbose,
     std::vector<unsigned char>* resultpng);
 
+#endif // __cplusplus
```

Comment:
Please two spaces before and end of line comment

Sorry for all those nit pick comments. If you're interested: this code is using the following style guide:

https://github.com/google/zopfli/pull/50/files#diff-460d0809311a0b9e58859036c51256fbR104